src/DependencyInjection/Configuration.php

```diff
@@ -446,6 +446,10 @@ private function addExtraSection(ArrayNodeDefinition $node): void
         $node
             ->children()
                 ->arrayNode('pixlr')
+                    ->setDeprecated(...$this->getBackwardCompatibleArgumentsForSetDeprecated(
+                        'The node "%node%" is deprecated and will be removed in version 4.0.',
+                        '3.28'
```

Comment:
```suggestion
                        '3.x'
```

Reply:
And should we add a NEXT_MAJOR comment ?

Reply:
Fixed.